src/client/Microsoft.Identity.Client/Instance/Discovery/RegionDiscoveryProvider.cs

```diff
@@ -14,6 +14,7 @@ internal class RegionDiscoveryProvider : IRegionDiscoveryProvider
     {
         private readonly IRegionManager _regionManager;
         public const string PublicEnvForRegional = "login.microsoft.com";
+        public const string PublicEnvForMtls = "mtlsauth.microsoft.com";
```

Comment:
I recommend you rename this to `RegionAndMtlsProvider`

Reply:
@bgavrilMS but the region is not included in the `const`?